cfgov/unprocessed/css/molecules/inset.less

```diff
@@ -78,12 +78,18 @@
   }
 
   &--email {
+    // This originally was a block,
+    // so we override the 30px margin added by .full-width-text-group .m-inset.
+    margin-bottom: unit(
+      @grid_gutter-width * 2 / @base-font-size-px,
+      em
+    ) !important;
```

Comment:
We could probably delete this, since regular `m-insets` don't have it, but I added it for now to retain the spacing at mobile.